doc/userGuide.md

```diff
@@ -97,3 +97,64 @@ For example, for this excerpt from a playlist:
 		...
 ```
 you will be able to run 'make scala_test' to execute the test.
+
+### Examples
+
+#### Sanity check an upstream JDK 8u patch
+
+Consider you are an upstream OpenJDK developer on Linux x86_64, and you'd like to run OpenJDK sanity tests locally on a patch for OpenJDK 8. Let the OpenJDK checkout be at `openjdk-jdk8u`, you've produced
+a *fastdebug* build and would like to run the regression test suite, `_sanity.openjdk` on it.
+
+```
+$ OPENJDK_SOURCES="$(pwd)/openjdk-jdk8u"
+$ OPENJDK_BUILD=$OPENJDK_SOURCES/build/linux-x86_64-normal-server-fastdebug/images/j2sdk-image
+$ tmpdir=$(mktemp -d)
+$ pushd $tmpdir
+$ git clone https://github.com/AdoptOpenJDK/openjdk-tests
+$ cd openjdk-tests
+$ TEST_DIR="openjdk_regression"
+$ export JAVA_BIN=$OPENJDK_BUILD/jre/bin
+$ export SPEC=linux_x86-64
+$ export JDK_IMPL=hotspot
+$ export BUILD_LIST=openjdk_regression
+$ TOP_DIR=$(pwd)
+$ export BUILD_ROOT=$TOP_DIR/openjdk_regression
+$ export JRE_IMAGE=$OPENJDK_BUILD/../j2re-image
```

Comment:
You do not need or use JRE_IMAGE in your example as it stands.

Reply:
Are you sure? My experiments showed that I needed to set this because as per [openjdk_regression.mk](https://github.com/jerboaa/openjdk-tests/blob/master/openjdk_regression/openjdk_regression.mk#L56) a default of `j2jre-image` will be used. OpenJDK builds only have `j2re-image`.

Reply:
Weird, as we should have same naming convention as OpenJDK builds.

Reply:
So are you saying that `j2jre-image` in `openjdk_regression.mk` should get fixed to `j2re-image`? My upstream JDK 8u build has this, fwiw:

    $ ls -1 build/linux-x86_64-normal-server-fastdebug/images/ | grep image
    j2re-image
    j2sdk-image